src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ImplicitNamespaceImports.targets

```diff
@@ -0,0 +1,22 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.ImplicitNamespaceImports.targets
+
+This file contains @(Import) items for C# 10 global usings.
+
+***********************************************************************************************
+-->
+<Project>
+  <!--
+    There's a master switch, $(DisableImplicitNamespaceImports),
+    which, if set, will prevent all implicit namespace imports.
+    This is done automatically, we don't have to check
+    $(DisableImplicitNamespaceImports) here.
+  -->
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_Android)' != 'true' ">
+    <Import Include="Android.App" />
+    <Import Include="Android.OS" />
+    <Import Include="Android.Runtime" />
```

Comment:
Another one that I'm not sure should be part of the default namespace set.  Sure, our default templates have used it for ages, but should it really be in the default `@(Import)` list?

I don't think excluding this namespace would break any of the templates in this PR.  Which suggests it's not *that* important.